sdks/python/scripts/run_pylint.sh

```diff
@@ -60,23 +60,34 @@ EXCLUDED_GENERATED_FILES=(
 apache_beam/portability/api/*pb2*.py
 )
 
+PYTHON_MAJOR=$(python -c 'import sys; print(sys.version_info[0])')
```

Comment:
Please add a JIRA to do lint checks for Python 3.

Reply:
done: https://issues.apache.org/jira/browse/BEAM-7884